### ISSUE TITLE
resample_branch not working with "folder structure" in trees

### ISSUE THREAD

**Issue by Kecksdose** [e5-tu-do/lhcb_pid_resample#3]
Unfortunately the function `resample_branch` is not working for a `TTree`, which has a folder kind structure like `MyDecayTree/DecayTree`. The error message I get is 

```
Traceback (most recent call last):
  File "pidtool.py", line 207, in <module>
    options.func(options)
  File "pidtool.py", line 169, in resample_branch
    for i, chunk in enumerate(read_root(options.source_file, ignore=["*_COV_"], chunksize=chunksize)):
  File "/home/tschmelzer/.local/lib/python2.7/site-packages/root_pandas/__init__.py", line 92, in read_root
    raise ValueError('More than one tree found in {}'.format(path))
ValueError: More than one tree found in /net/storage03/data/users/tschmelzer/NTuple/Bs2phiphi/MC/2011/MD/Bs2phiphi_MC_2011_MD.root
```

I fixed this by temporally changing the for-loop to 

``` python
for i, chunk in enumerate(read_root(options.source_file, "MyDecayTree/DecayTree", ignore=["*_COV_"], chunksize=chunksize))
```

**Comment by KonstantinSchubert** [comment]
Yes, that's a valid issue. Maybe we can offer to pass the tree as an additional option. Although I do not like that, as tree names are usually meaningless.

**Comment by KonstantinSchubert** [comment]
If I remember correctly, this  should be fixed with merge 91bf122. Otherwise please complain and I'll reopen.
